docs/user-guide/examples/advanced_examples/advanced_orchestration.md

```diff
@@ -1,4 +1,9 @@
-# Orchestration
+<!-- identify two options for testing this example:
+1. manual
+2. Merlin spec for Cosimulation
+ -->
+
+# Monte Carlo Co-Simulations
 
 ```{eval-rst}
```

Comment:
This creates an empty  toc (and there is a manually typed toc just below this), so I think the block here can be removed.